res/css/views/rooms/_RoomSublist.scss

```diff
@@ -27,52 +27,16 @@ limitations under the License.
         display: flex;
         align-items: center;
 
-        // ***************************
-        // Sticky Headers Start
-
-        // Ideally we'd be able to use `position: sticky; top: 0; bottom: 0;` on the
-        // headerContainer, however due to our layout concerns we actually have to
-        // calculate it manually so we can sticky things in the right places. We also
-        // target the headerText instead of the container to reduce jumps when scrolling,
-        // and to help hide the badges/other buttons that could appear on hover. This
-        // all works by ensuring the header text has a fixed height when sticky so the
-        // fixed height of the container can maintain the scroll position.
-
-        // The combined height must be set in the LeftPanel component for sticky headers
-        // to work correctly.
-        padding-bottom: 8px;
+        position: sticky;
+        top: 0;
```

Comment:
does this sticky both top and bottom like existing impl?

![image](https://user-images.githubusercontent.com/2403652/118633615-93264380-b7c9-11eb-8346-716e07f11fea.png)